src/test/unit_tests/test_namespace_fs.js

```diff
@@ -764,3 +1046,14 @@ async function read_object_stream(ns, bucket, file_key, dummy_object_sdk) {
 function create_file(file_path) {
     return fs.promises.appendFile(file_path, file_path + '\n');
 }
+
+async function create_object(object_path, data, version_id) {
+    const target_file = await nb_native().fs.open(DEFAULT_FS_CONFIG, object_path, 'w+');
+    await fs.promises.writeFile(object_path, data);
+    if (version_id !== 'null') {
+        const xattr_version_id = { 'user.version_id': `${version_id}` };
+        await target_file.replacexattr(DEFAULT_FS_CONFIG, xattr_version_id);
+    }
+
+    return target_file;
```

Comment:
no newline at the end of the file here and in test_bucketspace_versioning as well